projects/core/src/product/store/actions/product-search.action.ts

```diff
@@ -15,7 +15,7 @@ export const GET_PRODUCT_SUGGESTIONS_SUCCESS =
   '[Product] Get Product Suggestions Success';
 export const GET_PRODUCT_SUGGESTIONS_FAIL =
   '[Product] Get Product Suggestions Fail';
-export const CLEAN_PRODUCT_SEARCH = '[Product] Clean Product Search State';
+export const CLEAR_PRODUCT_SEARCH = '[Product] Clear Product Search State';
```

Comment:
Vere, very minor thing, but would consider making it aligned with action factory name:
`export const CLEAR_PRODUCT_SEARCH_RESULT = '[Product] Clear Product Search Result;`